Storage/storage.h

```diff
@@ -75,26 +75,6 @@ namespace storage {
 // 2) Splits the type into `T_ADDER` and `T_DELETER`, to support seamless persistence of deletions.
 // clang-format off
 
-#ifndef _MSC_VER
```

Comment:
BTW you made exactly the same change here:
https://github.com/dkorolev/Current/commit/f3756135e5539592556ffaa53477e74e9dcf6038
)))